lab-anthony/route/note-route.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const Note = require('../model/note');
+const response = require('../lib/response');
+const Storage = require('../lib/storage');
+
+const dataDir = new Storage(`${__dirname}/../data`);
```

Comment:
This is sort of a confusing variable name--it sounds like it's a filepath, not a storage manager.